### PR TITLE
reduce number of put operations on the user cluster

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
reduce number of put operations on the user cluster.
The missing ApiGroup led to diffs as it gets defaulted on the apiserver

**Release note**:
```release-note
NONE
```

### CODE REVIEW

**Comment by kdomanski** [comment]
What about https://github.com/kubermatic/kubermatic/blob/84319c87afb3bfc23e34a1c923777d0c42e3a223/api/pkg/resources/controllermanager/rolebinding.go ?

File: api/pkg/resources/controllermanager/clusterrolebinding.go [hunk 1]
```diff
@@ -23,9 +23,10 @@ func AdminClusterRoleBinding(data *resources.TemplateData, existing *rbacv1.Clus
 		APIGroup: "rbac.authorization.k8s.io",
```
Comment (kdomanski) [inline api/pkg/resources/controllermanager/clusterrolebinding.go]: `rbacv1.GroupName`?

Comment (mrIncompetent) [reply api/pkg/resources/controllermanager/clusterrolebinding.go]: Good catch, haven't seen that one

Comment (mrIncompetent) [reply api/pkg/resources/controllermanager/clusterrolebinding.go]: Actually there are quite a few more. I'll update all of them

**Comment by kdomanski** [comment]
https://github.com/kubermatic/kubermatic/blob/84319c87afb3bfc23e34a1c923777d0c42e3a223/api/pkg/resources/machinecontroler/clusterrolebinding.go#L59
`rbacv1.GroupName`?

**Comment by kdomanski** [comment]
https://github.com/kubermatic/kubermatic/blob/84319c87afb3bfc23e34a1c923777d0c42e3a223/api/pkg/resources/prometheus/rolebinding.go ?

**Comment by mrIncompetent** [comment]
With https://github.com/kubermatic/kubermatic/pull/1574/commits/2c14fb7f42f11843e80c7bc5ea4f7a375b22bba6 all got replaced by the const

**Comment by kdomanski** [comment]
https://github.com/kubermatic/kubermatic/blob/2c14fb7f42f11843e80c7bc5ea4f7a375b22bba6/api/pkg/resources/controllermanager/rolebinding.go#L41-L47

**Comment by kdomanski** [comment]
https://github.com/kubermatic/kubermatic/blob/84319c87afb3bfc23e34a1c923777d0c42e3a223/api/pkg/resources/prometheus/rolebinding.go#L28-L34

**Comment by mrIncompetent** [comment]
Last 2 missing Subjects covered by https://github.com/kubermatic/kubermatic/pull/1574/commits/8f99a1cf6e097e0110d97873c4e9f2c5fef2ee2f

**Review by kdomanski — APPROVED** [review]

**Pull request merged**